includes/reports/sales_snapshot.js

```diff
@@ -0,0 +1,80 @@
+module.exports = (params) => {
+  return publish("report_sales_snapshot", {
+    description: "Sales Snapshot report",
+    columns: {
+    // DIMENSIONS
+
+
+    // MEASURES
+    
+    },
+    assertions: {
+      //
+    },
+    ...params.defaultConfig
+  }).query(ctx => `
+
+    with opportunities_extended as (
+        select *
+        from ${ctx.ref("opportunities_extended")}
+    
+    ),
+    
+    pipeline as (
+
+    select 
+        round(sum(created_amount_this_month)) as pipeline_created_amount_this_month,
```

Comment:
What's the round for? Feels like we should avoid it if possible!

Reply:
This applies to quite a few places in the PR

Reply:
Agreed, best to let the analyst decide whether to round or not.